go.mod

```diff
@@ -1,13 +1,22 @@
 module gopkg.in/httprequest.v1
 
-go 1.15
+go 1.19
 
 require (
-	github.com/frankban/quicktest v1.10.0
-	github.com/google/go-cmp v0.4.0
+	github.com/frankban/quicktest v1.14.6
+	github.com/google/go-cmp v0.5.9
 	github.com/juju/qthttptest v0.1.1
 	github.com/julienschmidt/httprouter v1.3.0
-	golang.org/x/net v0.0.0-20200505041828-1ed23360d12c
+	golang.org/x/net v0.24.0
 	golang.org/x/tools v0.0.0-20200505023115-26f46d2f7ef8
```

Comment:
I suspect we could use a more recent non-pseudoversion version of this too.

Reply:
Agreed, done. I have also updated docs to clearly require 1.19.